src/app/charts/chart.component.ts

```diff
@@ -70,7 +70,14 @@ export class ChartComponent implements OnChanges {
     }
 
     onDownloadImageClicked() {
-        this.socialService.downloadAsPNG(this.chart.indicator.name);
+        let fileName: string = [
```

Comment:
In ES6 prefer template literal syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals